HackYourFuture-CPH/Guide-IT
# GET api route for all questions and questions by id

# Description

We want to be able to get questions with specific criteria so that we can show those questions. I created a GET request for all the questions and to retrieve a question by id. I'd like to add more specific criteria about searching for questions created before or after a certain date, and for searching for questions with specific words in. However, the issue instructed to break the PR's up a bit so that they aren't huge. I can add more once this has been approved.

Fixes # https://github.com/HackYourFuture-CPH/Guide-IT/issues/29

# How to test?

Once you have run the migration and seeding to your local database, open this branch and run npm run dev. Go to localhost:3000/api/documentation. You will see two GET requests for questions. You can run the tests from there, to retrieve all the questions and a question by its id.

# Checklist

- [x ] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [x ] This PR is ready to be merged and not breaking any other functionality
